doc/Customization-guide/Kaa-Server/Development-environment-setup/index.md

```diff
@@ -891,4 +890,4 @@ To provision your user verifier, do the following:
 
 1.  Place the user verifier descriptor and configuration classes into the Admin UI class path.
 2.  Place the user verifier implementation classes into the Operations Server class path.
-3.  Use [Admin UI999](https://docs.kaaproject.org/display/KAA/Administration+UI+guide#AdministrationUIguide-Userverifiers) or [REST API999](https://docs.kaaproject.org/display/KAA/Admin+REST+API#AdminRESTAPI-Create/edituserverifier) to create/update/delete your user verifier instances.
+3.  Use [Admin UI999](https://docs.kaaproject.org/display/KAA/Administration+UI+guide#AdministrationUIguide-Userverifiers) or [REST API999](https://docs.kaaproject.org/display/KAA/Admin+REST+API#AdminRESTAPI-Create/edituserverifier) to create/update/delete your user verifier instances.
```

Comment:
This guide is not yet ready and is a boilerplate. Links will be fixed in the jira for the guide.